src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -115,4 +115,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Warning,
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor UseHeaderDictionaryPropertiesInsteadOfIndexer = new(
```

Comment:
Can you open a matching issue on AspNetCore.Docs to track adding the documentation for this?

I think we should try to be proactive about documenting analyzers as we implement them for .NET 8 so we don't find ourselves having to write docs for multiple analyzes at the end of a release (like we are now 😅 ).

Reply:
https://github.com/dotnet/AspNetCore.Docs/issues/27081